### PR TITLE
Transaction Build integration tests

### PR DESCRIPTION
**Pull request by dkijania**
I've commented last assertion in test case:
test_correct_utxo_transaction_is_accepted_by_node

Because it's failing with the same error as #237

### CODE REVIEW

File: tests/common/configuration/node_config_model.rs [hunk 1]
```diff
@@ -48,9 +48,10 @@ impl NodeConfig {
     pub fn new() -> NodeConfig {
         let rest_port = get_available_port();
         let public_address_port = get_available_port();
+        let storage_suffix = rand::thread_rng().gen_range(1, 9999);
```
Comment (CodeSandwich) [inline tests/common/configuration/node_config_model.rs]: The more tests we add the more often they will collide and fail, right? ;)

Maybe we need a more serious tool, I've found this: https://crates.io/crates/tempfile, could it help?

File: tests/jcli_transaction_test.rs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        &second_utxo.out_addr,
+        &(second_utxo.out_value + first_utxo.out_value),
+    );
+
+    jcli_transaction_wrapper.assert_finalize_fail("not enough input for making transaction");
+}
+
+#[test]
+#[cfg(feature = "integration-test")]
+pub fn test_utxo_transation_with_more_than_one_witness_is_rejected() {
```
[the remark this answers is not in the document]
Comment (dkijania) [reply tests/jcli_transaction_test.rs]: Fixed

File: tests/jcli_transaction_test.rs [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+    jcli_wrapper::assert_post_transaction_default(&transaction_message);
+
+    let node_stats = jcli_wrapper::assert_rest_stats_command_default();
+    /*
+    assert_eq!(
+        "1",
+        node_stats.get("txRecvCnt").unwrap(),
+        "Node has not accepted transaction"
+    );*/
```
[the remark this answers is not in the document]
Comment (dkijania) [reply tests/jcli_transaction_test.rs]: Unfortunately we do. As i wrote in summary, this test is failing due to #237 issue. It need to stay like this until issue will be fixed.

File: tests/common/configuration/node_config_model.rs (hunk 1 above)
Comment (dkijania) [reply tests/common/configuration/node_config_model.rs]: Good point. fixed

File: tests/jcli_transaction_test.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (CodeSandwich) [reply tests/jcli_transaction_test.rs]: Aah, that's right. Then maybe we need a TODO comment with explanation and link to the issue before it? If I find this comment in 2 weeks, I won't know if I should uncomment, remove or keep it (just like now)

[the remark this answers is not in the document]
Comment (dkijania) [reply tests/jcli_transaction_test.rs]: Yes . I will do it.

**Pull request merged**